dockerfiles/bloodhound.Dockerfile

```diff
@@ -17,8 +17,8 @@
 ########
 # Global build args
 ################
-ARG SHARPHOUND_VERSION=v2.0.2
-ARG AZUREHOUND_VERSION=v2.1.6
+ARG SHARPHOUND_VERSION=v2.0.1
+ARG AZUREHOUND_VERSION=v2.1.5
```

Comment:
heads up - looks like you are missing some commits from main, unless this is intentional?